Makefile

```diff
@@ -33,7 +33,11 @@ rootfs-%: $(ROOTFS_BUILD_DEST)/.%_rootfs.done
 .PRECIOUS: $(ROOTFS_BUILD_DEST)/.%_rootfs.done
 $(ROOTFS_BUILD_DEST)/.%_rootfs.done:: rootfs-builder/%
 	@echo Creating rootfs for "$*"
-	$(ROOTFS_BUILDER) -o $(VERSION_COMMIT) -r $(ROOTFS_BUILD_DEST)/$*_rootfs $*
+	ifeq ("$*", "euleros")
+		$(ROOTFS_BUILDER) -o $(VERSION_COMMIT) -r $(ROOTFS_BUILD_DEST)/$*_rootfs $* || (>2 echo "ERROR: Could not generate euleros rootfs: see https://github.com/kata-containers/osbuilder/issues/46" && true)
```

Comment:
I'd also add a check for the `CI` env variable

Reply:
@devimc actually take a look to @jodh-intel comment: https://github.com/kata-containers/osbuilder/pull/236#issuecomment-471607426

Reply:
@jcvenegas ok, looks good